### PR TITLE
designed list and modal

### PR DESCRIPTION
**Pull request by itsmebipul00**

### CODE REVIEW

File: Components/Modal/Modal.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  color: hsl(var(--clr-dark));
+}
+.modal-fixed {
+  position: fixed;
+  inset: 0;
+  background-color: hsl(var(--clr-dark) / 0.7);
+  color: hsl(var(--clr-white));
+  padding: 4rem;
+  z-index: 99999;
+  backdrop-filter: blur(0.25rem);
```
Comment (Omkar-Ghate) [inline Components/Modal/Modal.css:24]: use px here?

**Review by Omkar-Ghate — APPROVED** [review]
Good to merge.

File: Components/Modal/Modal.css (hunk 1 above)
Comment (itsmebipul00) [reply Components/Modal/Modal.css:24]: rems are fine i guess, depends from person to person

**Review by sanu29 — APPROVED** [review]
Good to merge. Code is readable and clean. File structure is good. Naming has consistency.
👍

File: docs.css [hunk 2]
```diff
@@ -56,6 +62,10 @@ i {
   width: 70%;
 }
 
+.information > *:where(:not(:first-child)) {
+  margin-top: 1rem;
```
Comment (varun13169) [inline docs.css:66]: While using a complex selector it's preferable to add a comment explaining the logic.

**Review by varun13169 — APPROVED** [review]

**Pull request merged**